pulseapi/profiles/migrations/0014_bootstrap_relations.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2018-01-30 01:08
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def setup_default_values(apps, schema_editor):
+    ProfileType = apps.get_model('profiles', 'ProfileType')
+    ProfileType.objects.get_or_create(value='plain')
+    ProfileType.objects.get_or_create(value='staff')
+    ProfileType.objects.get_or_create(value='fellow')
+    ProfileType.objects.get_or_create(value='board member')
+    ProfileType.objects.get_or_create(value='grantee')
+
+    ProgramType = apps.get_model('profiles', 'ProgramType')
```

Comment:
note that if we want to bootstrap these properly, we'll need a list from @xmatthewx that can be used as starting point